Readability.js

```diff
@@ -1184,10 +1184,11 @@ Readability.prototype = {
     });
   },
 
-  _isEmptyElement: function(node) {
+  _isElementWithoutContent: function(node) {
     return node.nodeType === Node.ELEMENT_NODE &&
-      node.children.length == 0 &&
-      node.textContent.trim().length == 0;
+      node.textContent.trim().length == 0 &&
+      (node.children.length == 0 ||
+       node.children.length == node.getElementsByTagName("br").length + node.getElementsByTagName("hr").length);
```

Comment:
Hm, if this is sufficient I guess that works. In the future we may need to include other purely container elements - though maybe right now that already works? What happens for `<div><div><br></div></div>` ? Do we end up first removing the inner div and then remove the outer div afterwards, or something?